### PR TITLE
Update cli options format

### PR DESCRIPTION
**Pull request by Crizz0**
This will avoid, that there is written:
`–safe-mode`
it should be:
`--safe-mode`

### CODE REVIEW

File: development/cli/getting_started.rst [hunk 1]
```diff
@@ -64,14 +64,14 @@ Common options that can be used with any of phpBB's CLI commands.
...
-   --no-interaction (-n) | Do not ask any interactive question
-   --safe-mode | Run in Safe Mode (without extensions)
+   ``--help (-h)`` | Display a help message
+   ``--quiet (-q)`` | Do not output any message
+   ``--verbose (-v,-vv,-vvv)`` | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
+   ``--version (-V)`` | Display this application version
+   ``--ansi`` | Force ANSI (colors) output
+   ``--no-ansi`` | Disable ANSI (colors) output
+   ``--no-interaction (-n)`` | Do not ask any interactive question
+   ``--safe-mode`` | Run in Safe Mode (without extensions)
```
Comment (iMattPro) [inline development/cli/getting_started.rst:74]: Instead of parenthesis, use commas. For example:
```
``--verbose, -v, -vv, -vvv``
```

**Review by iMattPro — CHANGES_REQUESTED** [review]

File: development/cli/getting_started.rst (hunk 1 above)
Comment (Crizz0) [reply development/cli/getting_started.rst:74]: Both done.

File: development/cli/getting_started.rst [hunk 2]
```diff
@@ -64,14 +64,14 @@ Common options that can be used with any of phpBB's CLI commands.
...
-   --help (-h) | Display a help message
-   --quiet (-q) | Do not output any message
-   --verbose (-v,-vv,-vvv) | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
-   --version (-V) | Display this application version
-   --ansi | Force ANSI (colors) output
-   --no-ansi | Disable ANSI (colors) output
-   --no-interaction (-n) | Do not ask any interactive question
-   --safe-mode | Run in Safe Mode (without extensions)
+   ``--help, -h`` | Display a help message
+   ``--quiet,-q`` | Do not output any message
```
Comment (iMattPro) [inline development/cli/getting_started.rst:68]: Spacing

File: development/cli/getting_started.rst [hunk 3]
```diff
@@ -64,14 +64,14 @@ Common options that can be used with any of phpBB's CLI commands.
...
+   ``--verbose, -v,-vv,-vvv`` | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```
Comment (iMattPro) [inline development/cli/getting_started.rst:69]: Spaces

**Review by iMattPro — APPROVED** [review]

**Review by marc1706 — APPROVED** [review]

**Pull request merged**